roles/kubernetes/secrets/tasks/check-certs.yml

```diff
@@ -66,7 +70,7 @@
       {% set existing_certs = kubecert_master.files|map(attribute='path')|list|sort %}
       {% for cert in ['apiserver.pem', 'apiserver-key.pem', 'kube-scheduler.pem',
                       'kube-scheduler-key.pem', 'kube-controller-manager.pem',
-                      'kube-controller-manager-key.pem'] -%}
+                      'kube-controller-manager-key.pem','aggregator-proxy-client.pem','aggregator-proxy-client-key.pem'] -%}
```

Comment:
let's not make long lines when it can be helped. Can you move the aggregator-proxy-client-key.pem entry to the next line?